db/migrate/20150923125825_add_job_invocation_task_group.rb

```diff
@@ -0,0 +1,9 @@
+class AddJobInvocationTaskGroup < ActiveRecord::Migration
+  def up
+    add_column :job_invocations, :task_group_id, :integer, :index => true
```

Comment:
Yes. Should I add the database constraint or can we allow a job invocation to have no task group?